src/Documenter.jl

```diff
@@ -194,21 +194,24 @@ function makedocs(components...; debug = false, format = HTML(),
                   html_disable_git::Union{Bool, Nothing} = nothing, # deprecated
                   html_edit_branch::Union{String, Nothing} = nothing, # deprecated
                   html_canonical::Union{String, Nothing} = nothing, # deprecated
-                  kwargs...)
+                  assets::Union{Vector{T}, Nothing} = nothing, # deprecated
+                  analytics::Union{T, Nothing} = nothing, # deprecated
+                  kwargs...) where {T <: AbstractString, S <: AbstractString}
```

Comment:
```suggestion
                  kwargs...)
```

Reply:
Also, TIL that this doesn't error - `S` is not used in the signature.

Reply:
The `S` was supposed to be for the second argument, but what you suggested is much cleaner. Thanks!